src/parser.js

```diff
@@ -77,56 +130,29 @@ export function containIncludeCommand(commands = []) {
  */
 export function parseVariablesFromLabel(kvMap, label) {
     const kv = Object.assign({}, kvMap);
-    const beginEx = "^.*";
-    const endEx = ".*$";
-    const sepEx = ",?";
-    const kvsepEx = "[:=]";
-    const spacesEx = "\\s*";
-    const quotesEx = "[\"']";
-
-    Object.keys(kv).forEach(key => {
-        let keyEx = "(" + key + ")";
-        let valEx = "([-\\w\\s]*)";
-        if (key === "marker") {
-            keyEx = "(import|include)";
-            valEx = "(([-\\w\\s]*,?)*)";
+
+    let match = "";
+    while ((match = expressionRegExp.exec(label))) {
+        let key = match[1];
+        if (["include", "import"].includes(key)) {
```

Comment:
Shouldn't this be handled by babel instead of by us? I mean, you can use "babel-preset-env", and specify the node version in .babelrc

BTW, I was told that the gitbook server used node 5.1.*. But recently the broken gitbook-cli 2.3.1 moved to node LTS (currentlt v6).